pokemon-checker/src/DataTransferObjects/AbilityDTO.ts

```diff
@@ -0,0 +1,37 @@
+export type ListOfPokemon = {
+  name: string;
```

Comment:
Maybe someday we should move all these `name: string` and `url: string` patterns to be cast as a single object. I'm noticing that a lot of stuff (pokemon stub data, moves, abilities now) follows this name+URL pattern.

The individual types now are great because we know what type of name+URL pair we're dealing with, but I feel like we can make things easier than this someday in the future.

Reply:
Yeah its something I should have noticed earlier on, the api is broken up into different entities and every entity reference uses `name: string, url: string`, is this something you'd like to take a look at if you want to re-establish your knowledge of the app structure? Resolve this when you see/respond since it shouldn't be a blocker for the MR

Reply:
Created issue #65 and assigned it to you, if you don't want to take up on it (its not a huge rush, we can pushed it to the next milestone if need be), feel free to pass it to me